ansible/configs/ansible-automation-platform/roles/tower_validate_cfg/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+# tasks file for tower_validate
+- name: ping the Tower using the API
+  uri:
+    url: "https://{{ tower_hostname }}/api/v2/ping/"
+    validate_certs: "{{ tower_validate_certs }}"
+    method: GET
+    headers:
+      #Content-Type: "application/json"
+      Authorization: "Basic {{ (tower_admin_user + ':' + tower_admin_password) | b64encode }}"
```

Comment:
do we have to craft the headers? can't we use basic auth from the `uri` module directly ?

Reply:
Fixed with basic auth and enforcing basic auth.